cmd/soroban-rpc/docker/Makefile

```diff
@@ -0,0 +1,18 @@
+SUDO := $(shell docker version >/dev/null 2>&1 || echo "sudo")
+
+# https://github.com/opencontainers/image-spec/blob/master/annotations.md
+BUILD_DATE := $(shell date -u +%FT%TZ)
+
+TAG ?= stellar/stellar-soroban-rpc:$(SOROBAN_RPC_VERSION)
+
+STELLAR_CORE_VERSION ?= 19.10.1-1310.6649f5173.focal~soroban
+SOROBAN_RPC_VERSION ?= 0.8.0-21
+
+docker-build:
+	$(SUDO) docker build --pull $(DOCKER_OPTS) \
```

Comment:
are the deb pkgs published for both amd64 and arm64? If not arm64 yet, should probably set `--platform linux/amd64' , otherwise docker won't be able to build from any other non-amd hosts, like on dev workstations that are apple m1, etc.

Reply:
> are the deb pkgs published for both amd64 and arm64?

arm64 isn't published yet:
- https://github.com/stellar/packages/issues/154.

Reply:
> If not arm64 yet, should probably set `--platform linux/amd64' 
Good point, currently we only support amd64 builds, I added the flag